src/lib/homestore_backend/heap_chunk_selector.cpp

```diff
@@ -76,7 +76,7 @@ csharedChunk HeapChunkSelector::select_chunk(homestore::blk_count_t count, const
         return nullptr;
     }
 
-    const auto& vchunk = [it = std::move(it)]() {
+    const auto& vchunk = [it]() {
```

Comment:
Why not just:
```c++
auto vchunk = VChunk(nullptr);
auto& heap = it->second->m_heap;
if (auto heap_guard = std::lock_guard<std::mutex>(it->second->mtx); !heap.empty()) {
   vchunk = heap.top();
   heap.pop();
}
```
which avoids the allocation for the lambda?

Reply:
good suggestion and that make the code more simple.